docs/vectorizer/worker.md

```diff
@@ -87,94 +170,114 @@ On your local machine:
-       for a list of the available environment variables.
 
-    1. Run the vectorizer worker:
+## Advanced configuration options
 
-       ```shell
-       pgai vectorizer worker -d <db-connection-string>
-       ```
+Most users of the vectorizer worker will be happy with the default configuration for all other options.
+But, if you need to, you can control the following:
 
-## Run vectorizers with vectorizer worker
+- The vectorizer ids that are processed by the vectorizer worker ([section below](#control-which-vectorizers-are-processed))
+- The time between vectorizer worker runs ([section below](#set-the-time-between-vectorizer-worker-runs))
+- The number of asynchronous tasks running in a vectorizer worker ([section below](#set-the-number-of-asynchronous-tasks-running-in-a-vectorizer-worker))
+- Whether to run the vectorizer worker once and then exit ([section below](#run-the-vectorizer-worker-once-and-then-exit))
+
+All of these options can be set through the command line arguments, environment variables, or through an argument to the `Worker` 
+class constructor in the `pgai` Python package.
+
+| Option | Command line argument | Environment variable | `Worker` class constructor argument |
+|--------|-----------------------|--------------------|-------------------------------------|
+| Control which vectorizers are processed | `-i` / `--vectorizer-id` | `PGAI_VECTORIZER_WORKER_VECTORIZER_IDS` | `vectorizer_ids` |
+| Set the time between vectorizer worker runs | `--poll-interval` | `PGAI_VECTORIZER_WORKER_POLL_INTERVAL` | `poll_interval` |
+| Set the number of asynchronous tasks running in a vectorizer worker | `-c` / `--concurrency` | `PGAI_VECTORIZER_WORKER_CONCURRENCY` | `concurrency` |
+| Run the vectorizer worker once and then exit | `--once` | `PGAI_VECTORIZER_WORKER_ONCE` | `once` |
+
+
+### Control which vectorizers are processed
+
+If you want to run a vectorizer worker that only processes a subset of the vectorizers in your database,
+you can do so by specifying the vectorizer ids you want to process. You can do this by using the 
+`-i` / `--vectorizer-id` command line argument. 
 
-By default, when you run a vectorizer worker, it loops over the vectorizers defined in 
-your database and processes each vectorizer in turn. Five minutes after completing each 
-vectorizer run, the vectorizer worker loops over the vectorizers again. 
-For a [local installation](#install-vectorizer-worker-in-your-local-environment-), you use the 
-`-i` / `--vectorizer-id` command line argument to manage which vectorizers that are run by that
-worker instance. For `docker compose` you add arguments using either the `command` or `environment`
-flags in `compose.yaml`.
 
 A vectorizer worker can:
 
 - Run all vectorizers:
 
   To run all current and future vectorizers:
-  - local: `pgai vectorizer worker`
+  - cli: `pgai vectorizer worker`
+  - python: `worker = Worker(db_url=<your-database-connection-string>)`
```

Comment:
```suggestion
  - python: `worker = Worker(db_url="postgres://postgres:password@host:5432/database")`
```

I'd replace that everywhere with the example simply because it is one less look up to do at a user.